torch/csrc/utils/tensor_int_repr.cpp

```diff
@@ -0,0 +1,9 @@
+#include <torch/csrc/utils/tensor_int_repr.h>
+
+namespace torch { namespace utils {
+
+at::Tensor int_repr(at::Tensor t) {
```

Comment:
why do you need it as a separate function if you already have tensor method?

Reply:
Oh I think I have deleted this..